features/fixtures/maze_runner/ProjectSettings/ProjectSettings.asset

```diff
@@ -161,8 +161,11 @@ PlayerSettings:
   applicationIdentifier:
     Android: com.bugsnag.mazerunner
     iOS: com.bugsnag.unity.mazerunner
-  buildNumber: {}
-  AndroidBundleVersionCode: 1
+    iPhone: com.bugsnag.unity.mazerunner
+  buildNumber:
+    Standalone: 111
+    iOS: 111
+  AndroidBundleVersionCode: 111
```

Comment:
Use of the same number leaves the test open to some ambiguity. Can you use and assert different numbers for each of these? I'm not sure what `Standalone` is?

Reply:
Changed to different values per platform 👍 
Standalone is MacOS